.github/workflows/jekyll.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: Setup Pages
         id: pages
         uses: actions/configure-pages@v3
+      - run: git clone https://github.com/duckdb/duckdb
```

Comment:
I think this works just fine but maybe use the `checkout` Action? https://github.com/actions/checkout